src/plasmapy/particles/atomic.py

```diff
@@ -1143,27 +1143,33 @@ def _is_electron(arg: Any) -> bool:
 @particle_input
 @validate_quantities(energies=u.MeV)
 def stopping_power(
-    incident_particle: Particle,
+    incident_particle: ParticleLike,
```

Comment:
For `@particle_input`, we're switching to using annotations of `ParticleLike` instead of `Particle`.